omod/src/main/java/org/openmrs/module/owa/web/controller/OwaManageController.java

```diff
@@ -52,12 +52,21 @@
 	
 	@RequestMapping(value = "/deleteApp", method = RequestMethod.GET)
 	public String deleteApp(@RequestParam("appName") String appName, ModelMap model) {
-		if (appName != null && Context.hasPrivilege("Manage OWA")) {
```

Comment:
These two methods are confusing because whether deleting via REST or the Web UI, you are still deleting an owa app. Why not just add an optional parameter for the return url?

Reply:
I will look into it now